### DISCUSSION TITLE
Coding report problem - unable to select files

### DISCUSSION THREAD

**Discussion by ACR268** [ccbogel/QualCoder#787] [Q&A] [answered]
Hello, 

For some reason, I am now unable to run a coding report because none of my 16 files show up on the left where I can select them from. I was able to do it a few hours ago, but then I re-coded some of the data and when I went to run the report again, the files no longer show up.But they are definitely still in the project and all the coded data is there. 
I have tried closing, exiting and reopening, or even opening back-up versions of the project but nothing changes. 

Any suggestions please? 

![image](https://user-images.githubusercontent.com/129871991/229822510-a67f22ca-88b6-44b1-843e-8e28a710b724.png)

**Comment by ccbogel** [comment]
I did make some changes to the latest code (obtain via green code button, this could be a problem if you obtained and installed the latest code around this time), approximately 1 day ago, that had errors that were fixed about 12 hours later.
Try downloading the latest code again and installing and using again.
If this error still occurs, can you zip your project folder and send to me, so I can test and see where the problem is occurring.

**Comment by ACR268** [comment, marked as answer]
Solved in "Issues".
